api/src/service/domain/workflow/workflowitem_create.spec.ts

```diff
@@ -14,6 +14,13 @@ const address = "address";
 const root: ServiceUser = { id: "root", groups: [], address };
 const alice: ServiceUser = { id: "alice", groups: [], address };
 
+async function getUsersForIdentityStump(identity): Promise<string[]> {
+  if (identity === "alice") return ["alice"];
```

Comment:
@SamuelPull probably some code leftover for "alice" and "bob"?

Reply:
*.spec.ts is a test script, so probably alice and bob should be here :)